conda_build/post.py

```diff
@@ -1406,8 +1412,19 @@ def check_overlinking_impl(
                     list(diffs)[1:3],
                 )
                 sysroots_files[srs] = sysroot_files
+
+    def sysroot_matches_subdir(path):
+        if path.endswith("/"):
+            path = os.path.dirname(path)
```

Comment:
It looks like all of the paths will end with `/`, see line 1374 above:

```python
srs = sysroot if sysroot.endswith("/") else sysroot + "/"
```

So we can simplify this and remove the conditional